docs/api_reference.rst

```diff
@@ -663,6 +664,24 @@ Appears In:
    "``enabled`` (boolean)", "Enabled describes if Clowder should enable the private service and provide the configuration in the cdappconfig."
 
 
+.. _PrometheusConfig :
```

Comment:
The API docs are good. I think it's probably worth writing a blurb about this in the `docs/`. Perhaps as part of the usage stuffs. I forsee people asking us about this in the future as we onboard more apps.